lib/shiftzilla/org_data.rb

```diff
@@ -200,8 +228,14 @@ def generate_reports
           :releases        => [],
           :latest_snapshot => latest_snapshot,
           :all_bugs        => [],
+          :include_groups  => include_groups,
         }
 
+        # Check if this is actually a group "team"
+        if tname.start_with?("Group")
```

Comment:
This makes me think we need something better than string matching to positively identify a group. Maybe we add a boolean `is_group?` method to the `tinfo` object? I know that in the config validator you've added a test to prevent teams with a name that starts with 'Group' but it doesn't feel like a robust solution.

Reply:
I originally had that, but it only let me replace one or two of the string checks.  Its a bit weird, since we need to check for groups on the main page, but that `tinfo` is for `_overall`.  Its also a bit weird since one `populate_releases` ends up with a list of ordered team names - but not the objects themselves - so I ended up needing to check the group name using the starts_with to even set tinfo, which kind of negated the point.  I added a config check for teams that are named "Group*" to prevent issues.

I'll take a look tomorrow morning and see if I can add a is_group boolean to the teams.  I can't remember off the top of my head if I also need to be looking at the TeamData objects.